### ISSUE TITLE
Marks are very buggy in dirvish buffers

### ISSUE THREAD

**Issue by ViciousBadger** [justinmk/vim-dirvish#210]
They seem to move around on their own. That being said, I'm not sure a plugin like this _should_ be working with marks in the first place - but it would be very nice to be able to mark files, directories etc as a kind of vim-style bookmark system.

Steps to reproduce:
- Open dirvish
- Place a mark on some file, eg. by typing `mM`
- Browse around for a bit and head back to the directory where your mark is at
- The mark is now somewhere else, usually on the first line of the directory listing

**Comment by justinmk** [comment]
yeah, global marks kinda work but have the issue (at leaset) you mentioned. Would love it if they were more stable, but only willing to do this if there's a way to preserve them in bulk. Having tons of code trying to save/restore every mark is not an option.

**Comment by justinmk** [comment]
This seems to work a lot better now, no?
